ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -62,6 +62,21 @@ tests:
           cpu: "2"
           memory: 4Gi
     workflow: ipi-aws
+- as: e2e-console
+  run_if_changed: frontend/
+  steps:
+    cluster_profile: gcp-openshift-gce-devel-ci-2
+    test:
+    - as: console_test
```

Comment:
It's better to use the `-`, such as `console-test`

Reply:
updated